src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DateTimeOffsetList.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data;
+using Microsoft.Data.SqlClient.Server;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests.DataCommon
+{
+    public class DateTimeOffsetList : SqlDataRecord
+    {
+        public DateTimeOffsetList(DateTimeOffset dateTimeOffset)
+            : base(new SqlMetaData("dateTimeOffset", SqlDbType.DateTimeOffset, 0, 1))
+        {
+            this.SetValues(dateTimeOffset);
+        }
+    }
```

Comment:
If this class is only used within the current implementation, there's no need for it to be introduced in a separate file. It could be moved to the same file (`UdtDateTimeOffsetTest.cs`), considering it's a simple class in the test project with only one consumer.

Reply:
Move DateTimeOffsetList inside the unit test as suggessted.